janus_core/helpers/mlip_calculators.py

```diff
@@ -203,6 +206,27 @@ def choose_calculator(
 
         calculator = AlignnAtomwiseCalculator(path=path, device=device, **kwargs)
 
+    elif architecture == "sevennet":
+        from sevenn.sevennet_calculator import SevenNetCalculator
+
+        __version__ = "0.0.0"
+
+        if model_path is None or model_path.name == "":
```

Comment:
If I set `model_path = "."` this will also throw an error, I think

```
In [6]: x
Out[6]: PosixPath('.')

In [7]: x.name
Out[7]: ''
```

But you can imagine a user might want to do this, so I think we might need a different test.

I still think testing `if model_path == ""` within `_set_model_path` makes more sense, as this differentiates between "" and ".", and we can probably assume "" isn't valid for any MLIP.

Reply:
fixed by refactoring